lua/aerial/backends/util.lua

```diff
@@ -55,4 +55,59 @@ M.remove_change_watcher = function(bufnr, backend_name)
   })
 end
 
+-- Taken directly out of nvim-treesitter with minor adjustments
+if (vim.treesitter.language == nil) or (vim.treesitter.language.get_lang == nil) then
+  ---@param bufnr nil|integer
+  M.get_buf_lang = function(bufnr)
+    local ok, parsers = pcall(require, "nvim-treesitter.parsers")
+    if not ok then
+      return false, "nvim-treesitter not found"
+    end
```

Comment:
This guard statement and the pcall can be removed, since if we get this far we can assume that nvim-treesitter is installed.